File: Patterns/js/pattern6.js
/*
n = 4

4 3 2 1
4 3 2 1
4 3 2 1
4 3 2 1

*/

function pattern(n) {
  let ans = "";
  let row = 1;

  while (row <= n) {
    let col = 1;

    while (col <= n) {
      ans += n - col + 1 + " ";
      col++;
    }

    ans += "\n";
    row++;
  }

  console.log(ans);
}

pattern(5);
pattern(4);

File: Patterns/python/pattern6.py
# n = 4

# 1
# 2 3
# 4 5 6
# 7 8 9 10

n = int(input())
row = 1
val = 1
while row <= n:
    col = 1
    while col <= row:
        print(val, end="")
        col += 1
        val += 1
    row += 1
    print("")